pkg/querier/frontend/frontend.go

```diff
@@ -195,8 +199,13 @@ func (f *Frontend) handle(w http.ResponseWriter, r *http.Request) {
 		io.Copy(w, resp.Body)
 	}
 
-	// If LogQueriesLongerThan is set to <0 we log every query, if it is set to 0 query logging
-	// is disabled
+	f.reportSlowQuery(queryResponseTime, r, buf)
```

Comment:
Can we move the `if` condition outside, so that entire method isn't surrounded by it?

```suggestion
	if f.cfg.LogQueriesLongerThan != 0 && queryResponseTime > f.cfg.LogQueriesLongerThan {
		f.reportSlowQuery(queryResponseTime, r, buf)
	}
```

Reply:
I've left the if statement in metod with return early approach, if you prefer having if outside of the method I can change it later.